cumulus/parachains/runtimes/collectives/collectives-westend/src/lib.rs

```diff
@@ -0,0 +1,975 @@
+//! system where collators can reserve a small bond to join the block producer set. There is no
+//! slashing. Collective members are generally expected to run collators.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+#![recursion_limit = "256"]
+
+// Make the WASM binary available.
+#[cfg(feature = "std")]
+include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
+
+pub mod ambassador;
+pub mod impls;
+mod weights;
+pub mod xcm_config;
+// Fellowship configurations.
+pub mod fellowship;
+pub use ambassador::pallet_ambassador_origins;
+
+use cumulus_pallet_parachain_system::RelayNumberStrictlyIncreases;
+use cumulus_primitives_core::ParaId;
+use fellowship::{
+	migration::import_kusama_fellowship, pallet_fellowship_origins, Fellows,
+	FellowshipCollectiveInstance,
+};
+use impls::{AllianceProposalProvider, EqualOrGreatestRootCmp, ToParentTreasury};
+use sp_api::impl_runtime_apis;
+use sp_core::{crypto::KeyTypeId, OpaqueMetadata};
+use sp_runtime::{
+	create_runtime_str, generic, impl_opaque_keys,
+	traits::{AccountIdConversion, AccountIdLookup, BlakeTwo256, Block as BlockT},
+	transaction_validity::{TransactionSource, TransactionValidity},
+	ApplyExtrinsicResult, Perbill,
+};
+
+use sp_std::prelude::*;
+#[cfg(feature = "std")]
+use sp_version::NativeVersion;
+use sp_version::RuntimeVersion;
+
+use codec::{Decode, Encode, MaxEncodedLen};
+use frame_support::{
+	construct_runtime,
+	dispatch::DispatchClass,
+	genesis_builder_helper::{build_config, create_default_config},
+	parameter_types,
+	traits::{
+		fungible::HoldConsideration, ConstBool, ConstU16, ConstU32, ConstU64, ConstU8,
+		EitherOfDiverse, InstanceFilter, LinearStoragePrice,
+	},
+	weights::{ConstantMultiplier, Weight},
+	PalletId,
+};
+use frame_system::{
+	limits::{BlockLength, BlockWeights},
+	EnsureRoot,
+};
+pub use parachains_common as common;
+use parachains_common::{
+	impls::DealWithFees,
+	westend::{account::*, consensus::*, currency::*, fee::WeightToFee},
+	AccountId, AuraId, Balance, BlockNumber, Hash, Header, Nonce, Signature,
+	AVERAGE_ON_INITIALIZE_RATIO, DAYS, HOURS, MAXIMUM_BLOCK_WEIGHT, MINUTES, NORMAL_DISPATCH_RATIO,
+	SLOT_DURATION,
+};
+use sp_runtime::RuntimeDebug;
+use xcm_config::{GovernanceLocation, XcmConfig, XcmOriginToTransactDispatchOrigin};
+
+#[cfg(any(feature = "std", test))]
+pub use sp_runtime::BuildStorage;
+
+// Polkadot imports
+use pallet_xcm::{EnsureXcm, IsVoiceOfBody};
+use polkadot_runtime_common::{BlockHashCount, SlowAdjustingFeeUpdate};
+use xcm::latest::prelude::*;
+use xcm_executor::XcmExecutor;
+
+use weights::{BlockExecutionWeight, ExtrinsicBaseWeight, RocksDbWeight};
+
+impl_opaque_keys! {
+	pub struct SessionKeys {
+		pub aura: Aura,
+	}
+}
+
+#[sp_version::runtime_version]
+pub const VERSION: RuntimeVersion = RuntimeVersion {
+	spec_name: create_runtime_str!("collectives"),
+	impl_name: create_runtime_str!("collectives"),
+	authoring_version: 1,
+	spec_version: 10000,
```

Comment:
We should adopt the same versioning as the Fellowship runtimes. Since the next SDK release is 1.3.0, this should be `1_003_000`.
```suggestion
	spec_version: 1_003_000,
```